sdk/openai/azure-ai-openai/README.md

```diff
@@ -59,6 +59,24 @@ OpenAIAsyncClient client = new OpenAIClientBuilder()
     .buildAsyncClient();
 ```
 
+### Create a Non-Azure OpenAI client with API keys
+Get Non-Azure OpenAI API keys from [Open AI authentication API keys][non_azure_openai_authentication].
```

Comment:
I would be a bit more emphatic on the support to non-Azure OpenAI we are adding here. I am adding a suggestion. Feel free to disregard or re-word as you see fit :) 

```suggestion
### Support for non-Azure OpenAI

The SDK also supports operating against the public non-Azure OpenAI. The response models remain the same, only the setup of the `OpenAIClient` is slightly different. First, get Non-Azure OpenAI API key from [Open AI authentication API keys][non_azure_openai_authentication]. Then setup your `OpenAIClient` as follows: 

```